lib/Data/Content/CreateContentDraftData.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\RepositoryForms\Data\Content;
+
+use eZ\Publish\API\Repository\Values\ValueObject;
+
+class CreateContentDraftData extends ValueObject
+{
+    public $contentId;
```

Comment:
@bdunogier Symfony `PropertyPathMapper::mapFormsToData()` doesn't like that, it needs to write to the data.
http://api.symfony.com/3.1/Symfony/Component/Form/Extension/Core/DataMapper/PropertyPathMapper.html#method_mapFormsToData
Could workaround that with setter methods, but does it make sense?